parser.y

```diff
@@ -4463,12 +4466,28 @@ ExplainStmt:
 ExplainFormatType:
 	"TRADITIONAL"
 	{
-		$$ = "row"
+		$$ = "traditional"
```

Comment:
Does it need to be lowercase? If no, we can simply remove `$$ = "xxxx"`.

Reply:
Great, I remote it